frontend/src/metabase/admin/settings/components/widgets/EmbeddingSwitchWidget/EmbeddingSwitchWidget.tsx

```diff
@@ -0,0 +1,34 @@
+import { t } from "ttag";
+import * as MetabaseAnalytics from "metabase/lib/analytics";
+import { Stack, Switch, Text } from "metabase/ui";
+
+interface EmbeddingSwitchWidgetProps {
+  setting: { value: boolean | null };
+  onChange: (value: boolean) => void;
+}
+
+export const EmbeddingSwitchWidget = ({
+  setting,
+  onChange,
+}: EmbeddingSwitchWidgetProps) => (
+  <Stack spacing={"md"} className="text-measure">
+    <Text lh={1.5}>
+      {t`Embed dashboards, questions, or the entire Metabase app into your application. Integrate with your server code to create a secure environment, limited to specific users or organizations.`}
+    </Text>
+    <Switch
+      labelPosition="left"
+      checked={setting.value ?? false}
+      label={<strong>{t`Embedding Enabled`}</strong>}
+      onChange={e => {
+        const newValue = e.currentTarget.checked;
```

Comment:
TIL `currentTarget`. This is new to me. But is there any reason why you couldn't use `e.target.checked` in this case? `e.target` is used more across our codebase.

Reply:
Mmm I think I tried to switch it from target to currentTarget when I encountered an issue to see if it fixed it and forgot to change it back.
I'll change it back for consistency, thanks for pointing that out